ipaserver/plugins/topology.py

```diff
@@ -519,7 +519,6 @@ def execute(self, *keys, **options):
 
         return dict(
             result={
-                'in_order': not connect_errors and not max_agmts_errors,
```

Comment:
This breaks backwards compatibility with older (before this commit) clients. You can stop using the boolean in client code but you can't remove it from server reply.